lib/freight/apt.sh

```diff
@@ -454,5 +454,6 @@ apt_cache_source() {
 
 # Clean up old packages in the pool.
 apt_clean() {
-	find "$VARCACHE/pool" -links 1 -delete || true
+	find "$VARCACHE/pool" -links 1 -type f -delete
+	find "$VARCACHE/pool" -links 1 -type d -empty -delete
```

Comment:
`-links 1` should probably be removed from this search, I don't think it's too applicable for a dir.